airflow/kubernetes/pod_launcher.py

```diff
@@ -17,7 +17,6 @@
 # under the License.
 """
 This module is deprecated.
-
 Please use :mod:`kubernetes.client.models` for V1ResourceRequirements and Port.
```

Comment:
These should either be merged into one line or have an empty line between to conform to D205 checks